packages/wallets/src/wallets.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useSubscription } from '@deriv/api';
+
+const Wallets: React.FC = () => {
```

Comment:
Entry point component, I just rendered some stuff as an example so we can see something after enabling the flag.